mutagen-runner/src/main.rs

```diff
@@ -36,7 +37,9 @@ struct Options {
 fn run() -> Fallible<()> {
     let mutagen_start = Instant::now();
 
-    let opt = Options::from_args();
+    // drop mutagen arg in cargo-subcommand mode
```

Comment:
What if there is another "mutagen" arg?

Reply:
I've added code to try and detect when we're in "cargo subcommand" mode. [From the wiki](https://doc.rust-lang.org/cargo/reference/external-tools.html#custom-subcommands), `cargo` sets the `CARGO` env var to its path on the system. This is assuming that the user hasn't set the `CARGO` env var himself for some reason :/